code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -722,7 +720,7 @@
 	cyborg_base_icon = "synd_engi"
 	moduleselect_icon = "malf"
 	can_be_pushed = FALSE
-	magpulsing = TRUE
+	//module_traits = list(TRAIT_PUSHIMMUNE, TRAIT_NEGATES_GRAVITY)
```

Comment:
Do we want this commented out?

Reply:
There are some modules I have left commented out because they will damage the first testmerge with too many balance changes.